python/lsst/ts/electrometer/mock_server.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+import re
+import threading
+import queue
+
+from serial import Serial
+
+
+class ElectrometerSimulator:
+    def __init__(self, port):
+        self.port = port
+        self.log = logging.Logger(__name__)
+        self.buffer = queue.Queue(maxsize=50000)
+        self.command_calls = {"*IDN?": self.get_identification}
+        self.commands = [
+            re.compile(r"^(?P<cmd>\*IDN\?)$")
+        ]
```

Comment:
Please pick another instance name. You already use "commands" in the CSC to store an instance of `ElectrometerCommand` (though perhaps `command_factory` might work better for that if you rename it). 

It really confused me to find `commands` being used for something so different here.

How about `command_regexes`?

Also...I can appreciate the generality of a list, in case you start having to parse complicated commands, but is it really necessary? You clearly have managed without it. Do you expect to add more commands? If not, the code would be simpler if you just stored the one regex.

Reply:
I do expect to attempt to add more commands to this mock server. My hope is to have it be used for unit testing the CSC code and controller code. Setup is just is more involved as you have to create a virtual serial port for it with something like socat.